fetch_pyqt_windows.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
I try to minimize the number of files in the top-level. Would you mind moving this and the other new top-level files into `./contrib/win32`?  I think the last thing would then be updating the readme to list the new instructions for creating the installer (since I don't actually know those myself). I'll try and test it soon too in my windows env. and see how it works.

Is possible for `windows-installer.cfg` to live in the subdirectory too? I'm assuming the pynsist invocation probably has an option for specifying it, so I'll wait and see if that's reflected in the readme.

I agree the `--git-path` stuff can wait for later ~ I can always tweak that stuff once this is merged. Thanks for helping with this, I'm looking forward to using it.